src/select.tsx

```diff
@@ -52,7 +52,8 @@ const useStyles = createUseStyles((theme) => ({
         left: 12 + 1, // 12 for select paddingLeft and +1 for border
         right: 48 + 1, // 48 for icon and +1 for border
         overflow: 'hidden',
-        top: ({label}) => (label ? 27 : 17),
+        top: ({label}) => (label ? 26 : 16),
+        lineHeight: '18px',
```

Comment:
```suggestion
        lineHeight: '20px',
```
Rolling back your changes and change this line I think solve the problem and keep the same the distances between forms text and selects

![image](https://user-images.githubusercontent.com/6722153/155386723-66181655-70a7-4cc3-972f-0c7cc640c7d9.png)